### DISCUSSION TITLE
Nutritional deficiency diseases, may need to restructure

### DISCUSSION THREAD

**Discussion by laurenechan** [monarch-initiative/mondo#5370] [Ideas]
@nicolevasilevsky and I have talked briefly about proposing a pattern focused on nutritional deficiencies and excesses and their coordinated disease states. 
I was considering attempting to add existing diseases that are children of 'nutritional deficiency disease' to a pattern with a logical axiom like:

Ariboflavinosis
MONDO:0004573
Nutritional deficiency disease realized in response to some ‘decreased amount’ of ‘exposure to riboflavin via consumption'

While I think this example is a little clunky and likely not our best approach, any feedback on developing a pattern or more for nutritional exposures (or lack thereof) would be of great interest to me.

**Comment by laurenechan** [comment]
Some more thoughts. It appears to me we have a few different "buckets" of nutritional diseases...

General nutrient deficiencies: nondescript states of nutrient X deficiency that are caused by inadequate meeting of needs (specific to the organism in question). Focused on primary phenotypic presentation instead of behavior, insufficient to meet expected needs given the species and other related variables (lack of substance). There is no underlying biological state that is causing the deficiency. 
Example: niacin deficiency (not currently in Mondo)
- is this a disease state that we want in Mondo? Or maybe this is better suited for HPO?
- in the pattern, nutritional deficiency disease is_a deficiency of an essential nutrient w/o a specific phenotypic presentation
 
Specific nutrient deficiencies: descriptive diagnostic states of nutrient X deficiency that are caused by an inadequate intake compared to "average" or expected needs given the species and other related variables. These include distinct diagnostic markers or characterizing traits. There is no underlying biological state that is causing the deficiency, it is intake related. 
Example: pellagra (MONDO_0019975)
- may want to create these terms separately, outside of the pattern, just including the logical axiom to existing and adding new
 
Nutrient deficiencies caused by an abnormality: deficiencies to nutrient X caused by a failure at some level of digestion, absorption, and/or metabolism that lead to inadequacies of a nutrient. This can be instances of an increased nutrient needs that have a biological state leading to the increased nutrient needs/ inadequate intake status. This is a case of a biological status leading to the nutrient deficiency. 
- are there ~5-10 conditions that require the pattern? if yes, than can make pattern
 
Nutrient deficiencies that cause an abnormality: this may result from any of the above "buckets" , but includes specific metabolic ramifications that result from an insufficient quantity of nutrient X. 
 
Similar buckets can be made with nutrient intakes at a toxic level.

**Comment by nicolevasilevsky** [comment]
@laurenechan let me know if this needs discussion, we can discuss on the Mondo curation call

**Comment by nicolevasilevsky** [comment]
@laurenechan can we discuss this on a curation call? Can you please add it to the agenda when you can attend, thanks!
